### PR TITLE
Updates and additions to the changelog

### PR DESCRIPTION
**Pull request by justinegeffen**

### CODE REVIEW

File: changelog/seqera-enterprise/v23.4.0.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Tower Enterprise v23.4.0
+date: 2024-01-11
```
Comment (justinegeffen) [inline changelog/seqera-enterprise/v23.4.0.mdx:3]:
```suggestion
date: 2024-08-02
```

File: changelog/seqera-enterprise/v23.4.4.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Tower Enterprise v23.4.4
+date: 2024-04-21
```
Comment (justinegeffen) [inline changelog/seqera-enterprise/v23.4.4.mdx:3]:
```suggestion
date: 2024-03-20
```

File: changelog/seqera-enterprise/v23.4.5.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+title: Tower Enterprise v23.4.5
+date: 2024-04-21
```
Comment (justinegeffen) [inline changelog/seqera-enterprise/v23.4.5.mdx:3]:
```suggestion
date: 2024-07-15
```

File: changelog/seqera-enterprise/v23.4.6.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Tower Enterprise v23.4.6
+date: 2024-04-21
```
Comment (justinegeffen) [inline changelog/seqera-enterprise/v23.4.6.mdx:3]:
```suggestion
date: 2024-08-01
```

**Review by justinegeffen — COMMENTED** [review]
Updating dates

**Comment by justinegeffen** [comment]
@llewellyn-sl, I'd like to merge this PR this week. Could you build your 24.2 release notes on top of these changes?

File: changelog/seqera-cloud/v24.2.0_cycle25.mdx [hunk 5]
```diff
@@ -9,7 +9,7 @@ tags: [seqera cloud]
 - Quick launch now accepts pipelines that do not have a branch named `master`.
```
Comment (christopher-hakkaart) [inline changelog/seqera-cloud/v24.2.0_cycle25.mdx:9]:
```suggestion
- Quick Launch now accepts pipelines that do not have a branch named `master`.
```

File: changelog/seqera-enterprise/v22.3_major.mdx [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+## New features
+
+### Resource labels
+
+Tower now supports applying [resource labels](https://docs.seqera.io/platform/22.3/resource-labels/overview) to compute environments and other Tower elements. This offers a flexible tagging system for annotation and tracking of the cloud services consumed by a run.
+
+Resource labels are sent to the service provider for each cloud compute environment in `key=value` format. They can be created, edited, and applied by a workspace admin or owner.
+
+/Users/justine.geffen/docs/changelog/seqera-cloud/v24.1.0_cycle17.mdx
```
Comment (christopher-hakkaart) [inline changelog/seqera-enterprise/v22.3_major.mdx:19]: Your name included here

File: changelog/seqera-enterprise/v22.3_major.mdx [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+
+Tower now supports [Google Cloud Batch](https://docs.seqera.io/platform/22.3/compute-envs/google-cloud-batch) compute environments. Google Cloud Batch is a comprehensive cloud service suitable for multiple use cases, including HPC, AI/ML, and data processing. Tower now provides an integration with your existing Google Cloud account via the Batch API. While it is similar to the Google Cloud Life Sciences API, Google Cloud Batch offers a broader set of capabilities.
+
+Google Cloud Batch automatically provisions resources, manages capacity, and allows batch workloads to run at scale. The API has built-in support for data ingestion from Google Cloud Storage buckets. This makes data ingestion and sharing datasets efficient and reliable.
+
+This is a Beta Tower feature — more capability will be added as [Nextflow Google Cloud Batch support](https://www.nextflow.io/blog/2022/deploy-nextflow-pipelines-with-google-cloud-batch.html) evolves.
+
+### Admin panel enhancements
+
+The Tower [admin panel]../platform_versioned_docs/version-22.3/administration/overview.mdx) now provides additional user and organization management features.
```
Comment (christopher-hakkaart) [inline changelog/seqera-enterprise/v22.3_major.mdx:41]:
```suggestion
The Tower [admin panel](../platform_versioned_docs/version-22.3/administration/overview.mdx) now provides additional user and organization management features.
```

File: changelog/seqera-enterprise/v23.2_major.mdx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+## New features
+
+### AWS Fargate support
+
+Leveraging the Fusion file system, you can now run the Nextflow head job of your pipelines with the AWS Fargate container service. Fargate is a serverless compute engine compatible with Amazon ECS that enables users to run containers without the need to provision servers or clusters in advance. The scalable Fargate container service can help speed up pipeline launch and reduce cloud-related costs by minimizing the time for cloud infrastructure to be deployed.
+
+### Other improvements
+
+- Add support for Graviton architecture in AWS Batch compute environments
```
Comment (christopher-hakkaart) [inline changelog/seqera-enterprise/v23.2_major.mdx:15]: Some sections without full stops

**Review by christopher-hakkaart — CHANGES_REQUESTED** [review]
Very minor suggestions.

**Review by christopher-hakkaart — APPROVED** [review]
Looks good. Nothing blocking.

**Pull request merged**